cluster-sync/ephemeral_provider.sh

```diff
@@ -41,7 +42,7 @@ function configure_hpp() {
       ./cluster-up/ssh.sh "node$(printf "%02d" ${i})" "sudo mkdir -p /var/hpvolumes"
       ./cluster-up/ssh.sh "node$(printf "%02d" ${i})" "sudo chcon -t container_file_t -R /var/hpvolumes"
   done
-  HPP_RELEASE=$(curl -s https://github.com/kubevirt/hostpath-provisioner-operator/releases/latest | grep -o "v[0-9]\.[0-9]*\.[0-9]*")
+  HPP_RELEASE=v0.9.0
```

Comment:
I assume we want this hardcoded in 1.28 too, since it was like that in 1.34/38, @awels is that right?

Reply:
yes